.github/workflows/knative-stale.yaml

```diff
@@ -35,7 +35,7 @@ jobs:
           inactivity. Reopen the issue with `/reopen`. Mark the issue as
           fresh by adding the comment `/remove-lifecycle stale`.
         stale-issue-label: 'lifecycle/stale'
-        exempt-issue-labels: 'lifecycle/frozen'
+        exempt-issue-labels: 'lifecycle/frozen,triage/accepted'
```

Comment:
@lberk I know you're kind of the main person for Eventing triage. Now issues won't get stale if they're marked as triaged/accepted.